src/new/wallet.js

```diff
@@ -89,6 +89,7 @@ class HathorWallet extends EventEmitter {
     // Callback to be executed before reload data
     beforeReloadCallback = null,
     multisig = null,
+    preCalculatedAddresses = null,
```

Comment:
We need to add this to the docstring

Reply:
Fixed on [5c74a59](https://github.com/HathorNetwork/hathor-wallet-lib/pull/354/commits/5c74a595c99e414d85ee32b924b2159c0e12f5b7).

Also, refactored the whole constructor docstring to the _jsdocs_ format. It was a simple comment before.